llvm/lib/Transforms/Scalar/StraightLineStrengthReduce.cpp

```diff
@@ -267,9 +271,10 @@ FunctionPass *llvm::createStraightLineStrengthReducePass() {
 
 bool StraightLineStrengthReduce::isBasisFor(const Candidate &Basis,
                                             const Candidate &C) {
-  return (Basis.Ins != C.Ins && // skip the same instruction
-          // They must have the same type too. Basis.Base == C.Base doesn't
-          // guarantee their types are the same (PR23975).
+  return (Basis.Ins !=
+              C.Ins && // skip the same instruction
+                       // They must have the same type too. Basis.Base == C.Base
+                       // doesn't guarantee their types are the same (PR23975).
```

Comment:
Please delete this diff, then you're good to go!

Reply:
Sorted it sorry, happened when I used the `git clang-format HEAD~1` command

Reply:
Ahh, interesting. I didn't realize that's where the diff came from.